main.go

```diff
@@ -15,8 +15,45 @@ limitations under the License.
 */
 package main
 
-import "github.com/satesate-dev/go-restful-boilerplate/cmd"
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/satesate-dev/go-restful-boilerplate/cmd"
+	"github.com/satesate-dev/go-restful-boilerplate/helper/database"
+)
+
+func init() {
+
+	//Load .env
+	err := godotenv.Load("./.env")
+	if err != nil {
+		panic(err)
+	}
+}
 
 func main() {
 	cmd.Execute()
+
+	//Setup DB Connection
+	dbConfig := database.NewDatabase(
+		os.Getenv("DB"),
+		os.Getenv("DB_USERNAME"),
+		os.Getenv("DB_PASSWORD"),
+		os.Getenv("DB_HOST"),
+		os.Getenv("DB_PORT"),
+		os.Getenv("DB_NAME"),
+		os.Getenv("DB_TIMEZONE"),
+		os.Getenv("DB_SSL_MODE"),
+		os.Getenv("DB_SSL_CERT"),
+		os.Getenv("DB_SSL_KEY"),
+		os.Getenv("DB_SSL_ROOT_CERT"),
+	)
+
+	db, err := dbConfig.Connect()
+	if err != nil {
+		panic(err)
+	}
+
+	defer db.Close()
```

Comment:
move to function init db
example:
```
func initDatabase {
	dbConfig := database.NewDatabase(
  		os.Getenv("DB"),
  		os.Getenv("DB_USERNAME"),
  		os.Getenv("DB_PASSWORD"),
  		os.Getenv("DB_HOST"),
  		os.Getenv("DB_PORT"),
  		os.Getenv("DB_NAME"),
  		os.Getenv("DB_TIMEZONE"),
  		os.Getenv("DB_SSL_MODE"),
  		os.Getenv("DB_SSL_CERT"),
  		os.Getenv("DB_SSL_KEY"),
  		os.Getenv("DB_SSL_ROOT_CERT"),
  	)
 
  	db, err := dbConfig.Connect() -> change variable db to DBPool (make global variable put this var on the top of main file) = so this variable can be use for the rest handler -> model 
  	if err != nil {
  		panic(err)
  	}
 
  	defer db.Close()

}
```
then put the function init db into `cobra.OnInitialize`